yt_dlp/extractor/vlive.py

```diff
@@ -306,78 +308,93 @@ class VLiveChannelIE(VLiveBaseIE):
     }, {
         'url': 'https://www.vlive.tv/channel/FCD4B',
         'only_matching': True,
+    }, {
+        'url': 'https://www.vlive.tv/channel/FCD4B/board/3546',
+        'info_dict': {
+            'id': 'FCD4B',
+            'title': 'MAMAMOO - Star Board',
+        },
+        'playlist_mincount': 880
     }]
 
-    def _call_api(self, path, channel_key_suffix, channel_value, note, query):
-        q = {
-            'app_id': self._APP_ID,
-            'channel' + channel_key_suffix: channel_value,
-        }
-        q.update(query)
-        return self._download_json(
-            'http://api.vfan.vlive.tv/vproxy/channelplus/' + path,
-            channel_value, note='Downloading ' + note, query=q)['result']
-
     def _real_extract(self, url):
         channel_code = self._match_id(url)
 
-        channel_seq = self._call_api(
-            'decodeChannelCode', 'Code', channel_code,
-            'decode channel code', {})['channelSeq']
-
         channel_name = None
+
+        mobj = re.search(r'channel/[0-9A-Z]+/board/(?P<id>[0-9]+)', url)
+        if mobj:
+            board_id = mobj.group('id')
+            # check the board type
+            board = self._call_api(
+                'board/v1.0/board-%s', board_id, 'title,boardType')
+            board_name = board.get('title') or 'Unknown'
+            if board.get('boardType') not in ('STAR', 'VLIVE_PLUS'):
+                raise ExtractorError('Board "%s" is not supported' % board_name, expected=True)
+            posts_path = 'post/v1.0/board-%s/posts'
+            posts_id = board_id
+            query_add = {'limit': 100, 'sortType': 'LATEST'}
+        else:
+            board_name = None
+            posts_path = 'post/v1.0/channel-%s/starPosts'
+            posts_id = channel_code
+            query_add = {'limit': 100}
+
         entries = []
 
         for page_num in itertools.count(1):
+            # although the api has changed, I leave the following lines as notes
+
+            # video_list = self._call_api(
+            #     'getChannelVideoList', 'Seq', channel_seq,
+            #     'channel list page #%d' % page_num, {
+            #         # Large values of maxNumOfRows (~300 or above) may cause
+            #         # empty responses (see [1]), e.g. this happens for [2] that
+            #         # has more than 300 videos.
+            #         # 1. https://github.com/ytdl-org/youtube-dl/issues/13830
+            #         # 2. http://channels.vlive.tv/EDBF.
+            #         'maxNumOfRows': 100,
+            #         'pageNo': page_num
+            #     }
+            # )
+
```

Comment:
```
            #         # Large values of maxNumOfRows (~300 or above) may cause
            #         # empty responses (see [1]), e.g. this happens for [2] that
            #         # has more than 300 videos.
            #         # 1. https://github.com/ytdl-org/youtube-dl/issues/13830
            #         # 2. http://channels.vlive.tv/EDBF.
```

Modify this comment according to current API and move it beside relevent code (where `limit` is set, I believe)

Remove the rest